tools/CouchDB_Import_Demographics.php

```diff
@@ -287,10 +287,39 @@ function _generateQuery()
                             $cField.DateWithdrawn";
             }
         }
+
+        // Latest Diagnosis by project
+        $projects = \Utility::getProjectList();
+        foreach ($projects as $projectID => $project) {
+            $projectName  = str_replace(' ', '_', $project);
+            $latestProjDx = "latestDiagnosis_$projectName";
+
```

Comment:
To avoid error with projects that have a space in their name
```suggestion
            $projectAlias = \Project::getProjectFromID($projectID)->getAlias();
            $latestProjDx = "latestDiagnosis_$projectAlias";

```